### ISSUE TITLE
Wrong syntax highlighting with multiline string

### ISSUE THREAD

**Issue by valerianb** [Shopify/ruby-lsp#2404]
### Description

Hello, following VSCode's update to 1.92.0 (on Mac), I've got weird syntax highlighting with some multiline strings.

```ruby
some_string = <<-YAML
  id: "something"
YAML

def some_func
  "fhf"
end
```

Is highlighted as:

![Screenshot 2024-08-02 at 10 55 28](https://github.com/user-attachments/assets/bf0083d5-97fe-448a-9945-7222e0ac41b1)

Thanks

**Comment by valerianb** [comment]
Nope, sorry, forgot to close the issue. It works again as expected. Not sure if it's a VSCode or extension update that fixed it though.

Thanks

**Comment by andyw8** [comment]
Do you have another extension for YAML? Can you try disable that?

**Comment by valerianb** [comment]
Now with VSCode 1.94.2 and ruby lip 0.8.4 it is working fine again... Until the next update ? 🤔
